asn1scala/src/main/scala/asn1scala/asn1jvm_Bitstream.scala

```diff
@@ -1743,7 +2313,9 @@ case class BitStream private [asn1scala](
          v = wrappingExpr { (v & MASK_B(nBits)).toByte }
          currentBit = totalBitsForNextByte
       UByte.fromRaw(v)
-   }.ensuring(_ => buf == old(this).buf && BitStream.bitIndex(buf.length, currentByte, currentBit) == BitStream.bitIndex(old(this).buf.length, old(this).currentByte, old(this).currentBit) + nBits)
+   }.ensuring(_ => 
+      buf == old(this).buf 
+      && BitStream.bitIndex(buf.length, currentByte, currentBit) == BitStream.bitIndex(old(this).buf.length, old(this).currentByte, old(this).currentBit) + nBits)
 
```

Comment:
For this type of read functions, I'm not sure whether we can verify more. I think that the spec would be more convoluted than the function itself, which would not bring much more confidence.

Reply:
WDYT?

Reply:
I think we cannot verify more useful properties for read functions, as they are a bit like definitions.

Reply:
Perfect! We're on the same page then :)